src/pages/IndexPage.vue

```diff
@@ -1,16 +1,51 @@
 <template>
-  <q-page class="flex flex-center">
-    <h1>
-      <em class="fa-solid fa-person-digging"></em>
-      {{ $t('project.name') }}
-    </h1>
-  </q-page>
+  <div class="flex flex-center main">
+    <div class="fit row justify-center items-center">
+      <q-img
+        src="icons/logo_modelizer.svg"
+        width="144px"
+        height="174px"
+        :ratio="1"
+      />
+    </div>
+  </div>
+  <div class="flex flex-center">
+    <q-btn
+      color="primary"
+      size="xl"
+      icon="fa-solid fa-circle-plus"
+      :label="$t('project.create')"
+      :to="`/modelizer/${projectId}/model`"
+    />
+  </div>
 </template>
 
 <script>
```

Comment:
Following this [docs](https://vuejs.org/guide/extras/composition-api-faq.html#what-is-composition-api)
Please rewrite your script like this :
```
<script setup>
import (...);
const project = createProject();
</script>
```